### PR TITLE
added more examples to getUTCMonth() function

### LINKED ISSUE #4151: [Edit] Dates: Adding a codebyte for the entry `.getUTCMonth()`
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to the `getUTCMonth.md` entry. The term entry lives in docs/content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md.

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add a codebyte that shows how we can use `.getUTCMonth()`.
2) You can be creative with the codebyte snippet but make sure you don't deviate a lot from the entry.

### Codebyte Example

image

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Sahillather002**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

Closes #4151

### CODE REVIEW

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
The examples you added are fine but you were suppose to add codebytes also which are runnable code that readers can compile when reading your entry. In the end made a heading called Codebyte example and add running codebyte code with the tag ```codebyte/js.

**Comment by Sahillather002** [comment]
sure adding the codebyte

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md [hunk 1]
```diff
@@ -40,3 +41,46 @@ This results in the following output:
 11
 0
 ```
+
+## Codebyte Examples
+
+### Example with a specific date and formatting
+
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:48]: Add a small introductory sentence here above this codebyte which can give rough idea to the reader of what to expect from the code byte. Do the same for the cobytes below this too.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have done some changes from my side in this entry. Apart from that I have one comment for you to work on.

File: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md (hunk 1 above)
Comment (Sahillather002) [reply content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:48]: working on it...

Comment (Sahillather002) [reply content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md:48]: updated

**Comment by Sahillather002** [comment]
@dakshdeepHERE can pls confirm the changes

**Review by Sriparno08 — APPROVED** [review]
Looks good! 👍

**Pull request merged**